core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -1585,7 +1585,7 @@ class ReplicaManagerTest {
                                                  localLogOffset: Option[Long] = None,
                                                  offsetFromLeader: Long = 5,
                                                  leaderEpochFromLeader: Int = 3,
-                                                 extraProps: Properties = new Properties(),
+                                                 extraProps: Properties,
```

Comment:
why does not need to provide a default value to extraProps?

Reply:
I guess there are warnings because all code paths in test ignores the default value.

Reply:
+1 to Joseph's comment.